src/components/FollowersFollowing.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from "react";
+import { Button } from "antd";
+import FollowersFollowingUsers from "./FollowersFollowingUsers";
+import { useAppSelector } from "../redux/hooks";
+import { getProfile } from "../services/sdk";
+import { Profile } from "../utilities/types";
+import { ListStatus } from "../utilities/enums";
+
+const FollowersFollowing = (): JSX.Element => {
+  const graph = useAppSelector((state) => state.user.graph);
+  const [selectedListTitle, setSelectedListTitle] = useState<ListStatus>(
+    ListStatus.CLOSED
+  );
+
+  const [usersList, setUsersList] = useState<Profile[] | any>([]);
```

Comment:
its been a while since using Redux, does this belong in the Redux store?

Reply:
this is local to our component and its child so I think we can use state here.